services/skus/radom/radom.go

```diff
@@ -72,3 +74,67 @@ func (c *Client) CreateCheckoutSession(ctx context.Context, creq *CheckoutSessio
 
 	return resp, nil
 }
+
+func (c *Client) GetSubscription(ctx context.Context, subID uuid.UUID) (*SubscriptionResponse, error) {
+	req, err := c.client.NewRequest(ctx, http.MethodGet, "/subscription/"+subID.String(), nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Add("Authorization", c.authToken)
+
+	resp := &SubscriptionResponse{}
+	if _, err := c.client.Do(ctx, req, resp); err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+type SubscriptionResponse struct {
+	ID                string    `json:"id"`
+	NextBillingDateAt string    `json:"nextBillingDateAt"`
+	Payments          []Payment `json:"payments"`
+}
+
+type Payment struct {
+	Date string `json:"date"`
+}
+
+func (s *SubscriptionResponse) NextBillingDate() (time.Time, error) {
+	nxtB, err := time.Parse(time.RFC3339, s.NextBillingDateAt)
+	if err != nil {
+		return time.Time{}, err
+	}
+
+	return nxtB.UTC(), nil
+}
+
+const ErrPaymentsEmpty = Error("radom: payments is empty")
```

Comment:
nit: Would `Error("radom: empty payments")` also work, for brevity? Personally, and I might be wrong, when dealing with error strings and log messages, I use omit certain grammatical constructs (such as articles, tense and modal verbs).

Reply:
Sure. Reading this back both `payments is empty` and `empty payments` seem a little generic what about `radom: subscription payments empty`  which gives more context.

Reply:
Yes, I like that.